src/components/back-link/_back-link.scss

```diff
@@ -25,6 +25,11 @@
     // Underline is provided by a bottom border
     text-decoration: none;
 
+
+    &:focus {
```

Comment:
We may be able to close https://github.com/alphagov/govuk-frontend/issues/1236 with the focus state change here.